src/extractor/CaretValueRuleExtractor.ts

```diff
@@ -112,38 +112,65 @@ export class CaretValueRuleExtractor {
       if (flatParent[key] == null || !isEqual(flatSD[key], flatParent[key])) {
         const caretValueRule = new ExportableCaretValueRule('');
         caretValueRule.caretPath = key;
-        caretValueRule.value = getFSHValue(
-          key,
-          flatSD[key],
-          fisher.fishForFHIR('StructureDefinition', utils.Type.Resource),
-          fisher
-        );
+        caretValueRule.value = getFSHValue(key, flatSD[key], 'StructureDefinition', fisher);
         caretValueRules.push(caretValueRule);
       }
     });
 
     return caretValueRules;
   }
+
+  static processResource(
+    input: any,
+    fisher: utils.Fishable,
+    resourceType: 'ValueSet' | 'CodeSystem'
+  ): ExportableCaretValueRule[] {
+    const caretValueRules: ExportableCaretValueRule[] = [];
+    const flatVS = getPathValuePairs(input);
+    Object.keys(flatVS)
+      .filter(
+        key => !RESOURCE_IGNORED_PROPERTIES[resourceType].some(property => key.startsWith(property))
+      )
+      .forEach(key => {
+        const caretValueRule = new ExportableCaretValueRule('');
+        caretValueRule.caretPath = key;
+        caretValueRule.value = getFSHValue(key, flatVS[key], resourceType, fisher);
+        caretValueRules.push(caretValueRule);
+      });
+    return caretValueRules;
+  }
 }
 
-// The properties that are handled via keywords and other extractors
-const IGNORED_PROPERTIES = [
-  'resourceType',
-  'id',
-  'url', // NOTE: For now, we assume URL matches canonical and is generated
-  'name',
-  'title',
-  'description',
-  'fhirVersion',
-  'mapping',
-  'baseDefinition',
-  'derivation',
-  'snapshot',
-  'differential'
-];
+const RESOURCE_IGNORED_PROPERTIES = {
+  ValueSet: [
+    'resourceType',
+    'id',
+    'url',
+    'name',
+    'title',
+    'description',
+    'compose.include',
+    'compose.exclude'
+  ],
+  CodeSystem: ['resourceType', 'id', 'url', 'name', 'title', 'description', 'concept'],
+  StructureDefinition: [
+    'resourceType',
+    'id',
+    'url', // NOTE: For now, we assume URL matches canonical and is generated
```

Comment:
Now that we have a config in all cases, we probably shouldn't assume this anymore.  But that's beyond the scope of this PR.